pinc/LPage.inc

```diff
@@ -320,15 +320,15 @@ class LPage
         Page_markAsBad( $this->projectid, $this->imagefile, $this->round, $user, $reason );
 
         // Advise PM that the page has been marked bad.
-        $body_blurb =
-"Page {$this->imagefile} of this project has been marked bad due to {$PAGE_BADNESS_REASONS[$reason]}.
-Please visit
-    $code_url/tools/project_manager/handle_bad_page.php?projectid={$this->projectid}&image={$this->imagefile}
-to make any needed changes and make the page available for proofreading again.
-Until this report has been resolved, the project will not be able to leave
-the current round. If 10 pages are marked bad by at least 3 different users,
-the project will automatically be made unavailable.";
-        maybe_mail_project_manager($this->projectid, $body_blurb, "$site_abbreviation: Bad Page");
+        $body_blurb_messages[] = sprintf(_("Page %s of this project has been marked bad due to %s."), $this->imagefile, $PAGE_BADNESS_REASONS[$reason]);
+        $body_blurb_messages[] = _("Please visit");
```

Comment:
We should just combine this sentence into just one sprintf() string, probably with a translator's note so they know what it is:
```
// TRANSLATORS: %s is a URL
$body_blurb_messages[] = sprintf(_("Please visit %s to make any needed changes and make the page available for proofreading again.",
    "$code_url/tools/project_manager/handle_bad_page.php?projectid=$this->projectid&image=$this->imagefile");
```